services/api/lib/queries.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
-  findSkittleById: "SELECT id, color, user_id FROM skittles WHERE id = $1;",
-  createSkittle: "INSERT INTO skittles (color, user_id) VALUES($1, $2);",
-  updateSkittleById: "UPDATE skittles SET color = $1 WHERE id = $2",
-  deleteSkittleById: "DELETE FROM skittles WHERE id = $1"
+  users: {
```

Comment:
This is really neat. I like having all the SQL queries grouped together like this. :+1: